docs/en/latest/getting-started/load-balancing.md

```diff
@@ -44,6 +44,7 @@ curl -i "http://127.0.0.1:9180/apisix/admin/routes" -X PUT -d '
 You will receive an `HTTP/1.1 201 OK` response if the route was created successfully.
 
 :::info
+
 1. The `pass_host` field is set to `node` to pass the host header to the upstream.
 2. The `scheme` field is set to `https` to enable TLS when sending requests to the upstream.
 :::
```

Comment:
Need a blank here too?

Reply:
Yes, otherwise the CI will fail. However, I have discovered that all of the formatting errors in the documentation have been fixed by https://github.com/apache/apisix/pull/9379. Therefore, this PR can be closed